src/DataProtection/DataProtection/src/SimpleActivator.cs

```diff
@@ -13,6 +12,8 @@ namespace Microsoft.AspNetCore.DataProtection
     /// </summary>
     internal class SimpleActivator : IActivator
     {
+        private static readonly Type[] IServiceProviderTypeArray = { typeof(IServiceProvider) };
```

Comment:
nit: usually we use _ prefix naming for private fields, _serviceProviderTypeArray